### DISCUSSION TITLE
1.6.0

### DISCUSSION THREAD

**Discussion by yujincheng08** [LSPosed/LSPosed#1199] [New Release]
Happy Chinese National Day!
### Changelog
- Parasitize manager into Shell as it is a system app to avoid detecting manager package name as the manager is not installed on the system!  
- Fix crashes caused by some JIT info are recycled by JIT GC, thanks @2Y2s1mple
- Add a new API that allows deoptimizing some methods to fix some hooks not working caused by inlining, thanks @kdrag0n
- Fix modules may be loaded twice in Android 12 
- Fix rare inactivation after "soft reboot"
- Prevent crash when no browser is available
- Switching language is now available
- Optimize manager response

国庆节快乐！
### 更新日志
- 寄生管理器到 Shell，防止检测管理器包名
- 修复 JIT 信息被垃圾回收后引起的崩溃
- 添加新的 API 用于取消优化部分方法以避免 hook 在内联后失效
- 修复安卓 12 下模块可能加载两次的问题
- 修复软重启下低概率发生的未激活
- 修复禁用浏览器时管理器崩溃
- 支持切换语言
- 优化管理器响应

<hr /><em>This discussion was created from the release <a href='https://github.com/LSPosed/LSPosed/releases/tag/v1.6.0'>1.6.0</a>.</em>

**Comment by xsmiaos** [comment]
1.切换后台会自动软重启，打开某些软件有时候也会软重启
2.快捷方式那个管理器进入模块页面也不能看到模块，要刷新一下，点击模块，不能看到应用列表，而且全部都是关闭的，直接用程序管理正常

1.5.3 所有功能基本正常
Android8.1，api27，面具23，360n6手机

**Comment by Seasickil-CN** [comment]
国庆快乐
